ignite/contrib/handlers/wandb_logger.py

```diff
@@ -279,21 +276,12 @@ def __call__(self, engine: Engine, logger: WandBLogger, event_name: Union[str, E
                 " Please check the output of global_step_transform."
             )
 
-        metrics = self._setup_output_metrics(engine)
-        rendered_metrics = {}  # type: Dict[str, Union[str, float, numbers.Number]]
-        if self.tag is not None:
-            for name, value in metrics.items():
-                if isinstance(value, numbers.Number) or isinstance(value, str):
-                    rendered_metrics[f"{self.tag}/{name}"] = value
-                elif isinstance(value, torch.Tensor) and value.ndimension() == 0:
-                    rendered_metrics[f"{self.tag}/{name}"] = value.item()
-                elif isinstance(value, torch.Tensor) and value.ndimension() == 1:
-                    for i, v in enumerate(value):
-                        rendered_metrics[f"{self.tag}/{name}/{i}"] = v.item()
-                else:
-                    warnings.warn(f"WandBLogger output_handler can not log metrics value type {type(value)}")
-
-        logger.log(rendered_metrics, step=global_step, sync=self.sync)
+        rendered_metrics = self._setup_output_metrics(engine, log_text=True)
+        metrics = {}
+        for key, value in rendered_metrics.items():
+            metrics["/".join(key)] = value
```

Comment:
Actually, I just realized that we could introduce another flag like `log_text` and output keys either as tuples or as a string with "/". Such that we can simplify the code for TB, W&B and other similar loggers.

Reply:
Thats a good idea, I have added it